nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -85,6 +86,94 @@ def _apply_mask_and_get_affinity(seeds, niimg, radius, allow_overlap,
     return X, A
 
 
+def _get_spheres_rows(seeds, radius, allow_overlap, mask_img):
+    '''Utility function to get only the rows which are occupied by sphere at
```

Comment:
IIUC "rows" are boolean arrays that index a masked image, could this be stated a bit more explicitly (e.g. in the "returns" section)?